src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const oneDay = 40;
+  const dayPrice = oneDay * days;
+  let discount = 0;
+
+  if (days >= 0 && days <= 2) {
```

Comment:
The condition `days >= 0 && days <= 2` is correct for no discount. However, the logic for discounts should be checked for overlapping conditions.